.github/workflows/build.yml

```diff
@@ -89,10 +89,14 @@ jobs:
           CC: clang-3.9
           CXX: clang++-3.9
 
-      - name: Test
+      - name: Unittest
         run: |
           ./bin/minetest --run-unittests
 
+      - name: Integration test
+        run: |
+          ./util/test_multiplayer.sh
```

Comment:
I suggest adding a timeout. Not sure how long it takes to run, but I suggest taking that number and adding ~ 10%

```
- name: Integration test
  timeout-minutes: 5
  run: |
    ./util/test_multiplayer.sh
```

Reply:
10 seconds currently, the script should catch almost all timeout errors on it's own